src/components/alert/vl-alert.stories.js

```diff
@@ -0,0 +1,121 @@
+import { html } from "lit-html";
+import "../button";
+import "../alert";
+import styles from "./styles.scss";
+
+const defaultArgs = {
+  title: "Opgelet!",
+  icon: "warning",
+  content: html`<p>
+    U heeft geen rechten om deze actie uit te voeren.
+    <a href="#">Vraag rechten aan</a>.
+  </p> `,
+  size: "normal",
+  type: "info",
+  closable: false,
+};
+
+export default {
+  title: "custom-elements/vl-alert",
+  args: { ...defaultArgs },
```

Comment:
Heeft weinig zin ze hier te spreaden, om consequent te zijn kan je de props beter gewoon in het args object steken.